src/jit/gentree.h

```diff
@@ -1916,6 +1916,9 @@ class GenTreeUseEdgeIterator final
 #ifdef FEATURE_SIMD
     void MoveToNextSIMDUseEdge();
```

Comment:
UseEdges() is called in  lir.cpp so it needs the default parameter.
No caller ever passes false for this :-( so maybe we can eliminate it.